lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -204,15 +204,21 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 	auto stage = notice.GetStage();
 	for (const auto& changedPath : notice.GetResyncedPaths())
 	{
+		// When visibility is toggled for the first time or you add a xformop we enter
+		// here with a resync path. However the changedPath is not a prim path, so we
+		// don't care about it. In those cases, the changePath will contain something like:
+		//   "/<prim>.visibility"
+		//   "/<prim>.xformOp:translate"
+		if (!changedPath.IsPrimPath())
+			continue;
```

Comment:
At line 221 we specifically ask for the prim at path. So if this changed path is not a prim path we will get an invalid prim back.